site/content/3.13/data-science/arangographml/arangograph-ml.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ArangoGraphML Web Interface
+menuTitle: ArangoGraphML Web Interface
+weight: 15
+description: >-
+ Enterprise-ready, graph-powered machine learning as a cloud service or self-managed
+aliases:
+  - getting-started-with-arangographml
+---
+Solve high-computational graph problems with Graph Machine Learning. Apply ML on a selected graph to predict connections, get better product recommendations, classify nodes, and perform node embeddings. Configure and run the whole machine learning flow entirely in the web interface.
+
+## Creating a GraphML Project
+
+To create a new GraphML project using the ArangoDB Web Interface, follow these steps:
+
+- **Select the Target Database** – From the **Database** dropdown in the left-hand sidebar, select the database where the project should reside.
+- **Navigate to the Data Science Section** – In the left-hand navigation menu, click on Data Science to open the GraphML project management interface, then click on RunGraphML.
+![Navigate to Data Science](../../../images/datascience-intro.jpg)  
+- **Click "Add new project"** – In the **GraphML projects** view, click  **Add new project**.
+- **Fill in Project Details** – A modal titled **Create ML project** will appear. Enter a **name** for your machine learning project.
+- **Create the Project** – Click the **Create project** button to finalize the creation.
+- **Verify Project in the List** – After creation, the new project will appear in the list under **GraphML projects**. Click the project name to enter and begin creating ML jobs like Featurization, Training, Model Selection, Prediction.
+
+## Featurization Phase
+
+After clicking on a project name, you are taken to a screen where you can configure and start a new Featurization job. Follow these steps:
+- **Select a Graph** – In the **Features** section, choose your target graph from the **Select a graph** dropdown (Example, `imdb`).
+- **Choose Vertex Collections** – Pick the vertex collections (Example, `movie`, `person`) that you want to include for feature extraction.
+- **Select Attributes** – From the dropdown, choose the attributes from your vertex collection to convert into machine-understandable features. 
+
+{{< info >}}
+The following attributes cannot be used: imdb_feat_description, imdb_feat_genre, imdb_feat_homepage, imdb_feat_id, imdb_feat_imageUrl, imdb_feat_imdb_x_hash, imdb_feat_imdbId, imdb_feat_label, imdb_feat_language, imdb_feat_lastModified, imdb_feat_released, imdb_feat_releaseDate, imdb_feat_runtime, imdb_feat_studio, imdb_feat_tagline, imdb_feat_title, imdb_feat_trailer, imdb_feat_type, imdb_feat_version, imdb_x, imdb_y, prediction_model_output. As some of their values are lists or arrays.
+{{< /info >}}
```

Comment:
It's fine to mention that certain attributes are not eligible for GraphML but there shouldn't be a list of attributes here that are specific to the dataset, graph, and GraphML project. Users will not have these on the first run, and they will be different based on the mentioned things.